tests/vision_test.cpp

```diff
@@ -284,18 +283,13 @@ struct vision_test_case {
     }
 
     void test_all() const {
-        // Disabling 3d tests for now since 3d sight casting is actually
-        // different (it sees round corners more).
-        if( test_3d ) {
```

Comment:
The `if` can go, but the braces need to stay, so that only one `INFO` from this function is in scope for each test (that affects which show up in Catch2 failure reports).

Reply:
Fixed